README.md

```diff
@@ -54,64 +55,96 @@ var minioClient = new Minio.Client({
 
 ## Quick Start Example - File Uploader
 
-This example program connects to an object storage server, makes a bucket on the server and then uploads a file to the bucket.
+This sample ECMAScript module connects to an object storage server, creates a bucket, and uploads a file to the bucket.
```

Comment:
Perhaps this is an accurate description? Now that it's `.mjs`.

Reply:
as a script language it doesn't make much difference.